src/App.jsx

```diff
@@ -8,28 +8,92 @@ function App() {
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    new Promise((resolve, reject) => {
-      setTimeout(
-        () => resolve({data: {todoList: JSON.parse(localStorage.getItem('savedTodoList'))}}), 
-        2000
-      );
-    })
-    .then((result) => {
-      setTodoList(result.data.todoList);
+  async function fetchData(){
+    const options={};
```

Comment:
As an alternative, this object could be defined with its keys included like:
```
{
  method: 'GET',
  headers: {
    Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
  },
}
```

This is another one of those cases where the way you've created the object isn't wrong, but I thought it might be helpful to have the alternative method as well.